velox/benchmarks/tpch/TpchBenchmark.cpp

```diff
@@ -136,6 +140,10 @@ int main(int argc, char** argv) {
   } else {
     const auto queryPlan = queryBuilder->getQueryPlan(FLAGS_run_query_verbose);
     const auto task = benchmark.run(queryPlan);
+    // The Task can return results before the Driver is finished executing.
```

Comment:
This is a long comment and I wonder if it is practical to write it every time we use waitForTaskCompletion. Can we add documentation to waitForTaskCompletion function instead?